chemprop/train/make_predictions.py

```diff
@@ -92,8 +91,11 @@ def make_predictions(args: PredictArgs, smiles: List[List[str]] = None) -> List[
         num_workers=args.num_workers
     )
 
+    # Partial results for variance robust calculation.
+    all_preds = np.zeros((len(test_data), num_tasks, len(args.checkpoint_paths)))
```

Comment:
This is a big, likely unused, matrix. Maybe put creating it behind an args check for ensemble_variance.

Reply:
Good point, I added the check for args.ensemble_variance.